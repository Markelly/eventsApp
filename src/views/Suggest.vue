<template>
  <div class="suggest">
    <h1>SUGGEST</h1>
    <EventsList :isSuggest="true" />
  </div>
</template>

<script>
import EventsList from "../components/EventsList";
export default {
  name: "Suggest",
  components: { EventsList }
};
</script>

<style scoped lang="scss">
@import "../styles/lt-challenge";
</style>
