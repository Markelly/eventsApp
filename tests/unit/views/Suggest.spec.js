import Suggest from "../../../src/views/Suggest";
import { createLocalVue, shallowMount } from "@vue/test-utils";

describe("Cards", () => {
  const localVue = createLocalVue();
  let wrapper;

  test("should render the component", () => {
    wrapper = shallowMount(Suggest, { localVue });
    expect(wrapper).toMatchSnapshot();
  });
});
