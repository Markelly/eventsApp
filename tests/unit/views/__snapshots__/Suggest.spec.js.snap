// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards should render the component 1`] = `
<div class="suggest">
  <h1>SUGGEST</h1>
  <eventslist-stub issuggest="true"></eventslist-stub>
</div>
`;
